dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -6354,6 +6377,71 @@ definitions:
           format: "int64"
           readOnly: true
       description: "This describes one workflow in the dockstore"
+  Category:
```

Comment:
replace all occurence of `collection` with `category`

Reply:
I believe this is hard to do since Category is a subclass of Collection.